stable/redis/values.yaml

```diff
@@ -169,6 +169,13 @@ securityContext:
   enabled: true
   fsGroup: 1001
   runAsUser: 1001
+  ## sysctl settings for master and slave pods
```

Comment:
Can you add this block to the `stable-redis/values-production.yaml` file too?

Reply:
will do